### PR TITLE
CLI login updates

### PR DESCRIPTION
**Pull request by tjhiggins**

### CODE REVIEW

File: architect.yml [hunk 1]
```diff
@@ -119,6 +119,11 @@ services:
       GITHUB_CLIENT_ID: ${{ parameters.GITHUB_CLIENT_ID }}
       GITHUB_CLIENT_SECRET: ${{ parameters.GITHUB_CLIENT_SECRET }}
 
+      # expose_kratos_hack TODO remove
+      KRATOS: ${{ ingresses.kratos-public.url }}
+      HYDRA: ${{ ingresses.hydra-public.url }}
+      HYDRA_ADMIN: ${{ ingresses.hydra-admin.url }}
```
Comment (tjhiggins) [inline architect.yml:125]: Remove exposing of hydra admin

File: architect.yml [hunk 2]
```diff
@@ -161,6 +169,7 @@ services:
       POST_VERIFY_REDIRECT: ${{ parameters.POST_VERIFY_REDIRECT }}
       NUXT_ENV_LOGO_URL: ${{ parameters.LOGO_URL }}
       NUXT_ENV_APP_NAME: ${{ parameters.APP_NAME }}
+      ARCHITECT_CLOUD_APP_URL: '' # http://app.arc.localhost:1024 # TODO: update
```
Comment (ryan-cahill) [inline architect.yml:172]: @tjhiggins is there a decent way that I don't know of to get this value in a child component? I know we can get upstreams, but I've only found them in an array, so I can't select just the app by key

Comment (tjhiggins) [reply architect.yml:172]: What is it used for? I don't see it refed anywhere.

Comment (ryan-cahill) [reply architect.yml]: I found out this ended up being a duplicate of something else. It's gone now

**Comment by tjhiggins** [comment]
@ryan-cahill can you fix the preview and I'll merge
https://github.com/architect-team/architect-auth/runs/2760220337

**Pull request merged**